Mathlib/AlgebraicGeometry/ProjectiveSpectrum/StructureSheaf.lean

```diff
@@ -73,7 +73,7 @@ variable {𝒜}
 -/
 def IsFraction {U : Opens (ProjectiveSpectrum.top 𝒜)} (f : ∀ x : U, at x.1) : Prop :=
   ∃ (i : ℕ) (r s : 𝒜 i) (s_nin : ∀ x : U, s.1 ∉ x.1.asHomogeneousIdeal),
-    ∀ x : U, f x = Quotient.mk'' ⟨i, r, s, s_nin x⟩
+    ∀ x : U, f x = .mk ⟨i, r, s, by exact s_nin x⟩
```

Comment:
Do we understand why suddenly we need a `by exact`

Reply:
Turns out this is unnecessary.